src/civetweb.c

```diff
@@ -6079,6 +6090,10 @@ static char *addenv(struct cgi_env_block *block, const char *fmt, ...)
 	char *added;
 	va_list ap;
 
+	if (env == NULL) {
```

Comment:
The local variables here are `added` and `ap`, the parameters `block` and `fmt`.
There is no variable `env` in the code shown here.

There is an `env` parameter in the current code - but the `addenv` function has been completely rewritten between the version with the variables shown here (added, ap, fmt and block) and the current version (with variables n, space, truncated, added, ap, env and fmt).

I think this patch can no longer be applied to the current version.